app/models/conversation.rb

```diff
@@ -48,11 +49,11 @@ def latest_message
 
   def mark_notifications_as_read(user)
     notifications_as_conversation.where(recipient: user).unread.mark_as_read!
+    all_messages_read!
```

Comment:
Do we need to only mark the user's messages as read? If a a developer sends a message then opens the conversation again this code will mark it read even for the business.

Reply:
Ah, great catch. I'll add some logic for that as well.